### ISSUE TITLE
No longer runs on mybinder.org

### ISSUE THREAD

**Issue by betatim** [davidmascharka/tbd-nets#7]
Hey,

I was wondering if you tried running this lately and had an ideas as to why it doesn't run successfully anymore on mybinder.org. I don't really know much about the code in the repo nor pytorch. From looking at the `environment.yml` and the errors I get my guess would be that there is now a newer version of pytorch that changed conventions or some such?

I've used this repository before as an example in talks about Binder and wanted to do so again but during my run through I noticed that it doesn't work anymore. If you don't have time to fix this that is totally fine, I'll find a different repo for demo purposes.

**Comment by davidmascharka** [comment]
Thanks for catching this! I'm not sure what triggered a rebuild of the environment, but it was fetching newer versions of libraries that we can't make revert to the older versions and PyTorch did have some breaking changes in their more recent releases. I've updated the code in the Binder branch to be compatible with these latest versions, rebuilt the Binder environment, and was able to successfully run. If you could give this another go, let me know whether that fixed it for you. We're glad you've been able to use this in your talks!

**Comment by betatim** [comment]
We recently bumped the prefix we use to store docker images in mybinder.org because we made some changes to repo2docker (tool the does the building). That is probably the trigger of the rebuild.

Thanks for the fast fix! The full vqa notebook works but the one to reproduce the figures still gives an error when trying to index the answer. I think it needs an extra `tolist()` to turn a tensor into an int?
